packages/web/src/scss/components/Container/_theme.scss

```diff
@@ -1,6 +1,16 @@
 @use 'sass:map';
 @use '@tokens' as tokens;
+@use '../../tools/tokens' as tokens-tools;
+@use 'tools';
 
-$container-max-width: tokens.$container-max-width;
+$container-max-width: tokens-tools.get-variable-if-exists('container-xlarge-max-width', tokens.$container-max-width);
```

Comment:
Can you please add comments to all sections we will be able to remove when we remove the `container-max-width` token? 

Something that we would include the word `deprecat*` so we can fulltext search it later.

Maybe it can start `// @deprecated ...` so it is similar to JS deprecations.

Reply:
in all container files